gcloud/gcloud.go

```diff
@@ -0,0 +1,128 @@
+// Copyright © 2019 Percona, LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package gcloud
+
+import (
+	"fmt"
+	"math/rand"
+	"os"
+	"os/exec"
+	"strings"
+	"time"
+)
+
+type gcloud struct {
+	envName   string
+	project   string
+	zone      string
+	cluster   string
+	keyfile   string
+	namespace string
+}
+
+// New gcloud object
+func New(name string, config map[string]string) (*gcloud, error) {
```

Comment:
`go-lint` reports "exported func New returns unexported type *gcloud.gcloud, which can be annoying to use" and it is absolutely right

Reply:
done